NachoSpigot-Server/src/main/java/net/minecraft/server/EntityArrow.java

```diff
@@ -330,7 +330,7 @@ public void t_() {
                     if (this.shooter == null) {
                         world.makeSound(d, e, f, "random.bowhit", 1.0F, 1.2F / (this.random.nextFloat() * 0.2F + 0.9F));
                     } else {
-                        world.makeSound(this.shooter, "random.bowhit", 1.0F, 1.2F / (this.random.nextFloat() * 0.2F + 0.9F));
+                        world.makeSound(d, e, f, "random.bowhit", 1.0F, 1.2F / (this.random.nextFloat() * 0.2F + 0.9F));
```

Comment:
Same here

Reply:
Allright, give me a moment and i will fix that